data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
That's what I said, I removed the column but not the data.  Do you mean you
want the columns back?  I am starting to feel optimizing identical columns
is a lost battle.

On Fri, Aug 26, 2016 at 2:43 PM, Sergey Rubanov <notifications@github.com>
wrote:

> In data-non-standard.js
> <https://github.com/kangax/compat-table/pull/891#discussion_r76458461>:
>
> > @@ -1218,7 +1212,7 @@ exports.tests = [
> >      firefox12: false,
> >      firefox16: true,
> >      firefox17: false,
> > -    firefox20: true,
> > +    firefox20: false,
> >      firefox28: false,
>
> in fact there's a lot of firefox28 mentions in this file
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kangax/compat-table/pull/891/files/a35c2e4dd8c74a8732752e197bb497c094aa209f#r76458461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABRIeN2gw-nkadweAVf0giD6_ThMTFAWks5qjyW-gaJpZM4JsXuy>
> .
>